docs/manuals/source/Appendix/Howtos/StatisticsCollection.rst.inc

```diff
@@ -0,0 +1,36 @@
+.. _section-StatisticCollection:
```

Comment:
StatisticsCollection.rst.inc versus StatisticCollection section name: seems logical to fix one of them

Reply:
I have not changed in content, I justed splitted it into separate files. Anyhow, I've now adapted the section anchor.